turbinia/evidence.py

```diff
@@ -290,3 +290,11 @@ def __init__(self, text_data=None, *args, **kwargs):
     self.text_data = text_data
     super(ReportText, self).__init__(*args, **kwargs)
     self.copyable = True
+
+
+class TextFile(Evidence):
+  """Text data."""
+
+  def __init__(self, *args, **kwargs):
+    super(TextFile, self).__init__(*args, **kwargs)
+    self.copyable = True
```

Comment:
I don't remember what "copyable" means?

Reply:
You have seen it by now, but it's documented in https://github.com/google/turbinia/pull/209